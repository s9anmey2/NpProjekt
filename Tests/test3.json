{
	"epsilon":0.0000000001,
	"width":3,
	"height":3,
	"allGuardsAdded":true,
	"column2row2initialValue":{
		"0":{
			"0":1.0
		}
	},
	"guards":[
		[0,2,2,1,2],
		[0,1,1,1,2],
		[0,1,2,0,0],
		[1,0,1,0,2],
		[2,2,2,1,1],
		[2,2,2,2,2],
		[2,0,1,1,2],
		[3,0,2,0,1]
	],
	"commands":[
		[0.0,0.0,0.0,0.000009],
		[0.0,0.0,0.0,0.00001],
		[0.0,0.0,0.0,0.00001],
		[0.0,0.0,0.0,0.00001],
		[0.0,0.0,0.0,0.4230],
		[0.0,0.0,0.0,0.4700],
		[0.0,0.0,0.0,0.4700],
		[0.0,0.0,0.0,0.4700]
	]
}
