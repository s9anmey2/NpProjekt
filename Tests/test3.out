0.10843373493842585 0.10843373493861894 0.10843373493811885
0.10843373493842585 0.1084337349386189 0.12048192770902098
0.10843373493842585 0.1084337349386189 0.12048192770902091
